### PR TITLE
Refactor frontend router

### PR DESCRIPTION
**Pull request by vitormattos**
| ⚠️ |
|---|
| **ATTENTION!!** |

This PR was created from the branch of https://github.com/LibreSign/libresign/pull/2037

Before review this PR, is necessary merge https://github.com/LibreSign/libresign/pull/2037 to reduce the quantity of changes.

## Motivations of this PR

* Organize the frontend routes to follow the same pattern of other apps
* Fixes that was identified during manual tests
* Move part of logic from frontend to backend

### CODE REVIEW

File: lib/Controller/PageController.php [hunk 1]
```diff
@@ -122,21 +126,36 @@ public function indexFPath(): TemplateResponse {
 	 */
 	#[NoAdminRequired]
 	#[NoCSRFRequired]
+	#[RequireSetupOk]
```
Comment (giovannism20) [inline lib/Controller/PageController.php:129]: Why not main template here also ?

File: lib/Controller/PageController.php [hunk 2]
```diff
@@ -122,21 +126,36 @@ public function indexFPath(): TemplateResponse {
...
 	#[PublicPage]
 	#[RequireSignRequestUuid]
 	public function sign($uuid): TemplateResponse {
 		$this->initialState->provideInitialState('action', JSActions::ACTION_SIGN);
-		$this->initialState->provideInitialState('config', array_merge(
-			$this->accountService->getConfig($this->userSession->getUser()),
+		$this->initialState->provideInitialState('config',
```
Comment (giovannism20) [inline lib/Controller/PageController.php:134]: This code section are really confusing and in others areas that we are using for provide initial state. Let's consider to add some design patterns here to split this code.

File: lib/Controller/PageController.php (hunk 1 above)
Comment (vitormattos) [reply lib/Controller/PageController.php:129]: What about main template?
Can you explain more about this?

File: lib/Controller/PageController.php (hunk 2 above)
Comment (vitormattos) [reply lib/Controller/PageController.php:134]: Maybe after. At this moment we need to think more about if is necessary or not or also if we will split again the config.

For now the intention is only to reduce the responsibility of config method.

File: lib/Controller/PageController.php (hunk 1 above)
Comment (giovannism20) [reply lib/Controller/PageController.php:129]: Like this: `#[RequireSetupOk(template: 'main')]`

**Review by giovannism20 — APPROVED** [review]

**Pull request merged**